tests/test-translations.php

```diff
@@ -133,4 +133,18 @@ public function test_term_translations() {
 
 	}
 
+	public function test_canonical_content_fallback() {
+		$this->assertSame( 'en_US', get_locale() );
+
+		$en = $this->factory->post->create_and_get();
+		// In normal operation, the create_empty_translations method is called on an immediate single cron job
+		$GLOBALS['bbl_jobs']->create_empty_translations($en->ID);
+		$fr = bbl_get_post_in_lang( $en->ID, 'fr_FR', true );
+
+		// @FIXME: These tests fail due to the interaction of Babble_Post_Public::get_post_in_lang and Babble_Jobs::create_empty_translations
+		$this->assertSame( $en->post_title,   $fr->post_title );
```

Comment:
@joehoyle Originally fallback content was envisaged as more granular than that within a single post, so title, content, and excerpt can all fallback independently of each other: https://github.com/Automattic/babble/blob/3d211c82c1de853c08eda0fc70655d1bf189c6c9/class-post-public.php#L545-L571